pcdsdevices/sequencer.py

```diff
@@ -64,6 +67,13 @@ class EventSequencer(Device, MonitorFlyerMixin, FlyerInterface):
 
     def __init__(self, prefix, *, name=None, monitor_attrs=None, **kwargs):
         monitor_attrs = monitor_attrs or ['current_step', 'play_count']
+
+        # Setup Event Sequence
+        hutch_map = {1:'AMO', 2:'SXR', 3:'XPP', 4:'XCS', 5:'CXI', 6:'MEC', 7:'MFX'}
+        hutch = hutch_map[int(prefix[-1])]
+    
+        self.sequence = EventSequence('{}:ECS:IOC:01'.format(hutch), hutch_num=prefix[-1], name='{}_sequence'.format(hutch))
```

Comment:
Setting up `ophyd` subdevices like this is not recommended because it confuses some of the `bluesky` machinery. Every subdevice should be included as a `Component` in the class definition. However, there is no reason that this `EventSequence` object needs to be a `Device` subclass, so maybe we just nix that.